includes/modules/pages/checkout_payment/jscript_main.php

```diff
@@ -75,3 +75,4 @@ function doCollectsCardDataOnsite()
     });
 
 </script>
+<?php require(DIR_WS_MODULES . '/pages/checkout_confirmation/jscript_double_submit.php'); ?>

```

Comment:
Can we stick this require into a separate file? 
ie: since all jscript_*.php files are autoloaded, putting `jscript_double_submit.php` (or any name) into the checkout_payment folder, but still doing the require, makes code maintenance simpler.